src/archivematicaCommon/lib/archivesspace/client.py

```diff
@@ -345,17 +350,42 @@ def count_collections(self, search_pattern='', identifier=''):
 
         return self._get(self.repository + '/search', params=params).json()['total_hits']
 
-    def find_collections(self, search_pattern='', identifier='', fetched=0, page=1, page_size=30, sort_by=None):
+    def find_collections(self, search_pattern='', identifier='', accession='', fetched=0, page=1, page_size=30, sort_by=None):
+        """
+        Fetches a list of all resource IDs for every resource in the database.
+
+        :param string search_pattern: A search pattern to use in looking up resources by title or resourceid.
+            The search will match any title or resourceid containing this string;
+            for example, "text" will match "this title has this text in it".
+            If omitted, then all resources will be fetched.
+        :param string identifier: Restrict records to only those with this identifier.
+            This refers to the human-assigned record identifier, not the automatically generated internal ID.
+            This value can contain wildcards.
+        :param string accession: Restrict records to only those associated with this accession.
+            This should be provided using ArchivesSpace's internal format, for instance /repositories/2/accessions/1; it does not refer to human-assigned accession identifiers.
+            Use ArchivesSpaceClient.find_collections_by_accession to locate records via a human-assigned accession identifier.
+
+        :return list: A list containing every matched resource's ID.
```

Comment:
Sphinx doesn't support :return type: unfortunately.  It does support :return: and :returntype: http://sphinx-doc.org/domains.html#info-field-lists